src/containers/DebtPoolContainer/DebtPoolContainer.tsx

```diff
@@ -0,0 +1,34 @@
+import { Grid } from '@material-ui/core'
```

Comment:
uwaga co do folderu "containers" na przyszłość - powinniśmy w nim umieszczać komponenty połączone np. z danymi z Reduxa, którym nie trzeba już podawać skądś propów - acz myślę, że to można to chwilowo tu zostawić

Reply:
więc tu chwilowo nie musisz nic poprawiać czy przenosić